SubterfugeCore/Core/Generation/MapGenerator.cs

```diff
@@ -195,15 +199,17 @@ public void SetOutpostOwner(List<Outpost> outposts, Player player)
                     // If the current outpost is closer, put the current outpost in the list, replacing the farther outpost.
                     if(currentDistance < farthestDistance)
                     {
-                        closestOutposts[4] = o;
+                        closestOutposts[this.OutpostsPerPlayer - 1] = o;
                     }
                 }
             }
             
             // Once the closest outposts are determined, set the owner.
+            // And provide some drillers.
             foreach(Outpost closeOutposts in closestOutposts)
             {
                 closeOutposts.SetOwner(player);
+                closeOutposts.AddDrillers(30);
```

Comment:
- Shouldn't this be singular "closeOutpost" ? 
- The number of initial drillers should be defined in a template gamestate (to make it easier to impliment user defined game types?)? or is this temp value overwritten later on once the game is ready to start?

Reply:
Both points are correct, I'll update them.